server/handlers.go

```diff
@@ -147,8 +147,10 @@ func validateConfig(config *explorerlib.Config) error {
 func Stop(writter http.ResponseWriter, request *http.Request) {
 	Vars := mux.Vars(request)
 	id, _ := uuid.FromString(Vars["id"])
+	scenariolib.Info.Printf("Stopping worker")
```

Comment:
Please use `Println()`, `Printf()` is meant to be used with a format string.

`Println` will give you a new line every time.